release-lock-file-spec.md

```diff
@@ -90,117 +212,377 @@ Sources are declared in a key/value mapping.
 * Type: Object (String: String)
 
 
-### Contracts: `contracts`
+### Contract Types: `contract_types`
 
-The `contracts` field declares information about the deployed contracts
-included within this release.
+The `contract_types` field holds information about *contract types* which
+have been included in this release lockfile.  Release lockfiles **may not**
+include any *contract types* which are not present in the project's source
+code.
```

Comment:
Could do with a `.sol` example here. I assume you mean `contract C` would mean that the  type is C but from initial reading it's unclear.